pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.609.1</version>
```

Comment:
[CloudBees hat]
We should try to ensure that the parent is not updated beyond our oldest supported version unless there is a technical reason mandating the update
[/CloudBees hat]

My personal view is that I do not see any technical need for upping beyond 1.565 or 1.580 so I would prefer to keep one of those as the lower limit... Given that we are close to 1 year since 1.565 was cut I would be ok with 1.580... similarly if you can find a technical reason requiring the move above 1.580 I would be happy to consider, but just jumping straight to 1.609 without providing a good reason is not something I am happy with